code/modules/reagents/reagent_containers/food/snacks/cultural/human.dm

```diff
@@ -132,6 +132,27 @@
 	reagent_data = list(/singleton/reagent/nutriment = list("turkey" = 3))
 	bitesize = 3
 
+/obj/item/reagent_containers/food/snacks/sliceable/roast_chicken
+	name = "roast chicken"
+	desc = "Roasted and stuffed chicken surrounded by potatoes, all ready for the carving! Dibs on the drumsticks!"
+	icon = 'icons/obj/item/reagent_containers/food/cultural/human.dmi'
+	icon_state = "roast_chicken"
+	slice_path = /obj/item/reagent_containers/food/snacks/roast_chicken_slice
+	slices_num = 6
+	trash = /obj/item/tray/plate //Yes, this isn't the "trash" kind of plate. It's a big dish, so it's served on a large serving plate.
+	filling_color = "#9b5e2c"
+	reagents_to_add = list(/singleton/reagent/nutriment/protein = 24, /singleton/reagent/nutriment = 12, /singleton/reagent/soporific = 3)
```

Comment:
Why soporific?

Reply:
instead of triptophane, 'cus things like big old turkey meals make you tired, as I understand it. The Tofurkey also has it. I can toss it though if you want.

Reply:
might be fine if it doesn't put you too sleep